src/styles/utils/_variables.scss

```diff
@@ -1,4 +1,13 @@
-$days: "mon" 0, "tue" 1, "wed" 2, "thu" 3, "fri" 4, "sat" 5, "sun" 6;
-$cell-side: 100px;
-$gap: 1px;
-$calendar-padding: 10px;
+$container-day-width: 100px;
+$container-day-height: 100px;
+$day-gap: 1px;
+$calendar-width: 7 * $container-day-height + 20px + 6 * $day-gap;
```

Comment:
Here you should multiply 7 * $container-day-width instead, because if the width of the day would be changed, your calendar width will stay the same.